Sources/Kommunicate/Classes/KMConversationService.swift

```diff
@@ -346,6 +347,16 @@ public class KMConversationService: KMConservationServiceable, Localizable {
         if let teamId = conversation.teamId, !teamId.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty {
             metadata.setValue(teamId, forKey: ChannelMetadataKeys.teamId)
         }
+        
+        if let appName = conversation.appName {
+            let label = "iOS: \(appName)"
+            metadata.setValue(label, forKey: ChannelMetadataKeys.groupCreationURL)
+        } else {
+            if let appID = KMUserDefaultHandler.getApplicationKey(){
+                let label = "iOS: \(appID)"
+                metadata.setValue(label, forKey: ChannelMetadataKeys.groupCreationURL)
+            }
+        }
```

Comment:
combine this else & if let and rename label variable to meaningful. Use + operator to concatenate string